### PR TITLE
KEYCLOAK-5027 Add possibility to attach debugger to jboss based servers

### PR DESCRIPTION
**Pull request by vramik**
…server

### CODE REVIEW

File: testsuite/integration-arquillian/HOW-TO-RUN.md [hunk 1]
```diff
@@ -45,7 +45,21 @@ and adapter are all in the same JVM and you can debug them easily. If it is not
...
 and you will be able to attach remote debugger to the test. Unfortunately server and adapter are running in different JVMs, so this won't help to debug those. 
 
-TODO: Improve and add more info about Wildfly debugging...
+### JBoss auth server debugging
+
+When tests are run on JBoss based container (WildFly/EAP) there is maven profile which can be activated by 
+
+    -Pdebug-auth-server 
+
+By default the server will wait on startup to attach the debugger on localhost:5005. There are some properties what can change default behaviour.
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/HOW-TO-RUN.md]: Can we have the reverse logic employed here? That by default the server would _not_ wait?

File: testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml [hunk 2]
```diff
@@ -74,6 +74,7 @@
                 ${auth.server.feature}
             </property>
             <property name="javaVmArguments">
+                ${auth.server.jboss.jvm.args}
```
Comment (hmlnarik) [inline testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml]: maybe naming could reflect this parameter is used for debug, e.g. auth.server.jboss.jvm.debug.args ?

**Comment by vramik** [comment]
Thanks @hmlnarik, I've updated the PR according to your comments.

**Comment by hmlnarik** [comment]
Thank you, looks good to me

**Pull request merged**